### PR TITLE
Separate modifications from inputs in scenario data model

### PR DESCRIPTION
**Pull request by kdeloach**
Fix scenario data model so there is a clearer distinction between
modifications and inputs.

To test:
1. Run migrations
2. Try to add/remove/duplicate/delete/etc. scenarios

Existing scenarios probably won't work so I suggest clearing out your dev DB.

Connects #520

### CODE REVIEW

**Comment by mmcfarland** [comment]
I'm getting these same test failures locally.

**Comment by kdeloach** [comment]
Sorry about that, I failed to run the python unit tests before creating this PR. Will fix now.

**Comment by kdeloach** [comment]
Fixed tests and lint.

**Comment by mmcfarland** [comment]
After the fixups, I'm not able to add/display modifications any more, with errors generated from `updateModifications` in core/views.js.

**Comment by kdeloach** [comment]
I fixed a bug where modifications wouldn't appear on the map and squashed all fixups. Thanks for reviewing.

**Comment by mmcfarland** [comment]
+1 everything is still working and I see the precip input saved distinctly from modifications in the database.  Code improvements look good.

File: src/mmw/apps/modeling/tests.py [hunk 1]
```diff
@@ -15,16 +15,17 @@
...
         model_input = {
-            'modifications': [
+            'inputs': [
                 {
                     'name': 'precipitation',
                     'value': 1.2
                 }
             ],
-            'precip': 1.0
+            'modifications': []
```
Comment (caseypt) [inline src/mmw/apps/modeling/tests.py]: Just checking if we should be using 1.0 or 1.2 here.

Comment (caseypt) [inline src/mmw/apps/modeling/tests.py]: Ignore, didn't notice that this was a test.

**Comment by caseypt** [comment]
I don't think this causing problems, but I think at some point we should create a separate model and collection type for inputs. At the moment, they are saved with the default values for modifications:

selection_063

**Comment by caseypt** [comment]
+1. Tested. Nice job.

File: src/mmw/js/src/modeling/modificationConfigUtils.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ var modificationConfig = require('./modificationConfig.json');
 function getHumanReadableName(modKey) {
     if (modificationConfig[modKey]) {
         return modificationConfig[modKey].name;
-    } else {
-        throw 'Unknown Land Cover or Conservation Practice: ' + modKey;
     }
+    console.warn('Unknown Land Cover or Conservation Practice: ' + modKey);
```
Comment (lliss) [inline src/mmw/js/src/modeling/modificationConfigUtils.js]: This is sensible.

File: src/mmw/js/src/modeling/filters.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+var nunjucks = require('nunjucks'),
```
Comment (lliss) [inline src/mmw/js/src/modeling/filters.js]: Much cleaner.

**Comment by lliss** [comment]
+1. Tests pass. Found no errors or unexpected behaviour while testing.

**Pull request merged**